src/sentry/models/broadcast.py

```diff
@@ -16,13 +16,15 @@ class Broadcast(Model):
     __relocation_scope__ = RelocationScope.Excluded
 
     upstream_id = models.CharField(max_length=32, null=True, blank=True)
-    title = models.CharField(max_length=32)
+    title = models.CharField(max_length=64)
     message = models.CharField(max_length=256)
     link = models.URLField(null=True, blank=True)
     is_active = models.BooleanField(default=True, db_index=True)
     date_expires = models.DateTimeField(default=default_expiration, null=True, blank=True)
     date_added = models.DateTimeField(default=timezone.now)
     cta = models.CharField(max_length=256, null=True, blank=True)
+    media_url = models.URLField(null=True, blank=True)
+    category = models.CharField(max_length=32, null=True, blank=True)
```

Comment:
I don't think this should be `null` but either one of the defined categories

Reply:
Are we able to backfill existing rows to have a category if we make this not null?

Reply:
Yeah, I would not add a not null constraint here tbh - we can enforce the usage of categories in the frontend and during validation for new entries, adding this on a DB level just makes migration harder for no reason & I don't think we can programmatically backfill this